get_dependencies.py

```diff
@@ -24,18 +24,40 @@ def download(url, filename):
     sys.stdout.write('\n')
 
 
-print('[*] Extracting webdriver archives...')
-for f in ['geckodriver', 'nightly', 'chromedriver', 'chrome-linux']:
-    tar = tarfile.open('tools/%s.tar.xz' % f, 'r:xz')
-    tar.extractall(path='tools/')
-    tar.close()
+if _platform == "linux" or _platform == "linux2":  # linux
+    osys = "linux"
+elif _platform == "darwin":    # MAC OS X
+    osys = "mac"
+
+
+if osys == "linux":
+    print('linux os detected !')
+    print('[*] Downloading support files for linux ~ 108 mb')
+    download('https://www.dropbox.com/s/3mosdy9tsf7pilh/linux.zip?dl=1', 'linux.zip')
+    print('[*] Extracting linux.zip...')
+    with ZipFile('linux.zip', 'r') as z:
+        z.extractall("linux")
+    print('[*] Extracting webdriver archives...')
+    for f in ['geckodriver', 'nightly', 'chromedriver', 'chrome-linux']:
+        tar = tarfile.open('linux/%s.tar.xz' % f, 'r:xz')
+        tar.extractall(path='tools/')
+        tar.close()
+    os.remove('linux.zip')
+    os.system("rm -rf linux ")
+
+elif osys == "mac":
+    print('mac os detected !')
+    print('[*] Downloading support files for mac ~ 150 mb')
+    download('https://www.dropbox.com/s/k1szymmktj0e1pf/mac.zip?dl=1', 'mac.zip')
+    print('[*] Extracting mac.zip...')
+    os.system("unzip mac.zip -d tools")
+    os.system("cp tools/geckodriver" + " /usr/local/bin/ ")
```

Comment:
There are several problems here:
1. We should avoid os.system and use Python standard functions;
2. Why concatenating two strings here? Why not writing the entire string directly?
3. We should avoid writing any file to `/usr`.

Reply:
1: using python standard functions extracts the file but files get corrupted so used os.system.
2:  yes entire string can be passed directly.
3:  in mac it need to be copied in that directory.

Reply:
Why does it need to be in /usr/local/bin?

Reply:
3. i tested it now and it is actually not required. i did because  earlier i was facing the compability problem so i thought what if put those files into those directory and it started working. i will remove it.